CMakeLists.txt

```diff
@@ -343,7 +343,6 @@ if (INTERN_BUILD_MOBILE AND NOT BUILD_CAFFE2_MOBILE)
   set(FEATURE_TORCH_MOBILE ON)
   set(NO_API ON)
   set(USE_FBGEMM OFF)
-  set(USE_PYTORCH_QNNPACK ON)
   set(USE_QNNPACK OFF)
```

Comment:
Oh, I think I remember why I added the line above - because I turned USE_QNNPACK OFF and turned USE_PYTORCH_QNNPACK ON - I want to pair them up. i.e. the reason we turned off USE_QNNPACK here is because we turned on better alternative. (just command, no action needed)